src/plugins/solvers/linear_solver.cpp

```diff
@@ -74,7 +80,84 @@ namespace phylanx { namespace execution_tree { namespace primitives
         "Cholesky (LLH) decomposition. If uplo = 'L', solve "
         "`a` as a lower triangular matrix, otherwise as an upper "
         "triangular matrix."
-        )};
+        ),
+        PHYLANX_LIN_MATCH_DATA("iterative_solver_conjugate_gradient",
+         "a, b\n"
+         "Args:\n"
+         "\n"
+         "    a (matrix) : a matrix\n"
+         "    b (vector) : a vector\n"
+         "\n"
+         "Returns:\n"
+         "\n"
+         "A matrix `x` such that `a x = b`, solved using the "
+         "conjugate gradient"
+        ),
+        PHYLANX_LIN_MATCH_DATA("iterative_solver_BiCGSTAB",
```

Comment:
Could we please stick to our naming convetions we use in HPX/Phylanx (see: https://github.com/STEllAR-GROUP/hpx/wiki/HPX-Source-Code-Structure-and-Coding-Standards)?